test/core/sup_tree_core/periodic_log_writer_test.exs

```diff
@@ -1,11 +1,11 @@
-defmodule AntikytheraCore.ReductionLogWriterTest do
+defmodule AntikytheraCore.PeriodicLog.ReductionBuilderTest do
```

Comment:
Shouldn't this file be `test/core/sup_tree_core/periodic_log/reduction_builder_test.exs` to match the source directory?

Reply:
Fixed. c8b46353c875d65f392bf24d640193ee71256c4d